src/controllers/AuthController.ts

```diff
@@ -62,3 +62,39 @@ export const login = async (
     next(exception);
   }
 };
+
+export const checkLogin = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const authHelper = new AuthHelper();
```

Comment:
It is preferred to put this variable outside the function in the top level (global scope), this way you can use where ever you want and you reduce the times of its creation so the CPU does not have to create it again every time he calls this fnc.

Reply:
You are absolutely right here it was a mistake of me, thanks.
There is a better way for AuthHelper function is to make them static, since we don't need make instances of this class and it only consists of functions, so I have done that!